localnet/evmos/setup_genesis.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+CHAINID="evmos_9999-1"
+MONIKER="orchestrator"
+
+# Orchestrator account
+KEY="orchestrator"
+MNEMONIC="stumble tilt business detect father ticket major inner awake jeans name vibrant tribe pause crunch sad wine muscle hidden pumpkin inject segment rocket silver"
+
+BUILD_DIR=$(pwd)/localnet/build
+DATA_DIR=$BUILD_DIR/node4/evmosd
+GENESIS=$DATA_DIR/config/genesis.json
+TEMP_GENESIS=$DATA_DIR/config/tmp_genesis.json
+CONFIG=$DATA_DIR/config/config.toml
+
+# create necessary directory for orchestrator node
+mkdir -r $DATA_DIR
+
+echo "create and add new keys"
+echo $MNEMONIC | evmosd keys add $KEY --home $DATA_DIR --no-backup --chain-id $CHAINID --keyring-backend test --recover
+echo "init Evmos with moniker=$MONIKER and chain-id=$CHAINID"
+evmosd init $MONIKER --chain-id $CHAINID --home $DATA_DIR
+
+echo "Prepare genesis..."
+echo "- Set gas limit in genesis"
+cat $GENESIS | jq '.consensus_params["block"]["max_gas"]="10000000"' > $TEMP_GENESIS && mv $TEMP_GENESIS $GENESIS
```

Comment:
`shellcheck` would mark this as an unnecessary `cat` operation
```suggestion
jq '.consensus_params["block"]["max_gas"]="10000000"' "$GENESIS" > "$TEMP_GENESIS" && mv "$TEMP_GENESIS" "$GENESIS"
```